### DISCUSSION TITLE
__orchantiforgery_<version> cookie is not set as Secure

### DISCUSSION THREAD

**Discussion by miguelisidoro** [OrchardCMS/OrchardCore#15337] [Q&A]
Hi,

I am working on a solution that uses Orchard Core and I am solving possible security vulnerabilities.

One of the issues, I noticed is that the __orchantiforgery_version cookie is not set as Secure.

![image](https://github.com/OrchardCMS/OrchardCore/assets/5429152/e95325cb-ea43-471b-b278-b15fdd2cf68a)

Is there any way to set this cookie as Secure programmatically in our solution?

Thanks

**Comment by Skrypt** [comment]
I believe this is an option on the antiforgery cookie.

```c#
        services.AddAntiforgery(options =>
        {
            options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
        });
```

See https://github.com/OrchardCMS/OrchardCore/pull/15341

**Reply by miguelisidoro** [reply]
Thanks! It worked!
